frontend/src/components/dropdown.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+<li @click="visible = !visible;" class="dropdown-main-item"><a class="dropdown-icon"><img src="~../assets/menu.svg" class="dropdown-icon"></a>
```

Comment:
I don't see darkmode in the dropdown - will it still be a feature/will it be integrated later after merging this PR?

Reply:
This removes dark mode, it can be included later if need be, but in it's existing state dark mode didn't add a lot. It can definitely be added in a later PR if people deem it necessary.

Reply:
Had a conversation with @garoller today—dark mode currently has some deficiencies. I think ripping it out here makes sense, and then we should have a conversation about its future.

Reply:
Ya I agree, I think we should remove dark mode in this PR.